.travis.yml

```diff
@@ -21,11 +21,7 @@ services:
   
 jdk:
   - oraclejdk8
-install: mvn clean install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
 script: mvn --batch-mode clean verify | egrep -v "(^\[INFO\] Download|^\[INFO\].*skipping)"
```

Comment:
It looks like it's filtering out some of the noise from maven resolving dependencies.

This was already in the travis config so I originally thought I'd leave it, though now that I look at it again I don't think it really makes sense - I'd rather just have the full log output available and we can grep through it later if we need to. I'll remove it.